CMakeLists.txt

```diff
@@ -15,12 +15,8 @@
 # limitations under the License.
 # ~~~
 cmake_minimum_required(VERSION 3.17.2)
-list(APPEND CMAKE_MODULE_PATH "${CMAKE_CURRENT_SOURCE_DIR}/cmake")
 
-project(Vulkan-Loader)
-
-# Gets the header version used during code generation
-include(cmake/generated_header_version.cmake)
+project(VULKAN_LOADER VERSION 1.3.240)
```

Comment:
I changed the project name since it affects the project version variable.

IE:

`VULKAN_LOADER_VERSION` reads better than `Vulkan-Loader_VERSION`

Reply:
As long as that doesn't affect any of the install/exporting names that others might depend on, this should be good.

Reply:
The loader.pc.in has `@CMAKE_PROJECT_NAME@` which makes me wonder if that is the only place it'll be affected.

Reply:
> The loader.pc.in has @CMAKE_PROJECT_NAME@ which makes me wonder if that is the only place it'll be affected.

Good catch

> As long as that doesn't affect any of the install/exporting names that others might depend on, this should be good.

I don't see anything else that would be affected